### DISCUSSION TITLE
Can't use reaction emojis in review threads on mobile

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#17315] [Pull Requests]
Using Chrome for Android on a Pixel 3A, there's no reaction widget next to a review comment and there's no reaction option in the menu:

![Screenshot_20220526-104724](https://user-images.githubusercontent.com/2119212/170513749-35f6cab9-1e1c-40db-8fc9-e2c8f4219794.png)

If someone has already reacted to a comment, then the reaction widget is available.

**Comment by rooby** [comment]
This happens in issues as well as PRs.

**Comment by Avasam** [comment]
This post seems to mention the same thing. May as well go upvote it: https://github.com/community/community/discussions/12781

**Reply by jsoref** [reply]
In principle, that discussion is misfiled:

https://github.com/community/community/discussions/categories/mobile
<img width="787" alt="image" src="https://user-images.githubusercontent.com/2119212/212500746-23656ae0-a264-4b42-a044-14e657042fbf.png">
[GitHub Mobile](https://github.com/mobile)

**Comment by Djfe** [comment]
finally, we have the option now
